package.json

```
@@ -25,6 +25,7 @@
     "test:services": "npx npm-run-all --npm-path npm --print-label --parallel test:121-service",
     "test:121-service": "cd services/ && docker compose exec 121-service npm test",
     "test:portal": "npm test --prefix interfaces/Portal/",
+    "fix:prettier": "npm run test:prettier -- -w",
```

```suggestion
    "fix:prettier": "npm run test:prettier -- --write",
```

As a self-documenting feature, I'd like to use the "human-readable" versions of these command-line flags. (see the other scripts/commands above).